x-pack/plugins/apm/public/components/app/correlations/error_correlations.tsx

```diff
@@ -175,7 +173,7 @@ export function ErrorCorrelations({ onClose }: Props) {
             }
             status={correlationsStatus}
             setSelectedSignificantTerm={setSelectedSignificantTerm}
```

Comment:
Is the "significant terms" terminology still valid?

Reply:
We're also working on this in #108441, see the item `Some refactor of the Correlation table ...` in that PR over there.